### ISSUE TITLE
Custom position without enable it

### ISSUE THREAD

**Issue by neno1978** [evilgoldfish/LockGlyphX#35]
If I take the glitch whit the finger and position in some place in lock screen , position change and custom position enable himself . If I not enable it , it musn't enable for himself, if I not enable it, i musn't can pick the glitch and positionate it

**Comment by Sticktron** [comment]
I think this is by design. If you move the glyph with your finger it is just a shortcut to setting the custom position settings, except they are automatically filled in for you. Then you can fine-tune the number manually.

Sent from my iPhone

On Mar 5, 2017, at 10:16 AM, neno1978 <notifications@github.com<mailto:notifications@github.com>> wrote:


If I take the glitch whit the finger and position in some place in lock screen , position change and custom position enable himself . If I not enable it , it musn't enable for himself, if I not enable it, i musn't can pick the glitch and positionate it

**Comment by neno1978** [comment]
Yeah, but if you do not want this to happen, this happens. Imagine that there are notifications in the lockscreen, if you are going to move them you can catch the glitch without wanting and activate this without being your desire

**Comment by Sticktron** [comment]
So the real issue is accidentally moving the glyph? That can be addressed :)

Sent from my iPhone

On Mar 5, 2017, at 11:06 AM, neno1978 <notifications@github.com<mailto:notifications@github.com>> wrote:


Yeah, but if you do not want this to happen, this happens. Imagine that there are notifications in the lockscreen, if you are going to move them you can catch the glitch without wanting and activate this without being your desire

**Comment by Sticktron** [comment]
So the issue is accidentally moving the glyph. We can do something about that.

**Comment by neno1978** [comment]
Yeah! Exactly , the inssue is  accidentally move the glyph ;)

**Comment by AppleBetas** [comment]
I'm not quite sure but maybe what he is saying is that if custom position is disabled, you shouldn't be able to drag it around

**Comment by AppleBetas** [comment]
Either way, that still makes it accidentally movable for people that needed to move it, so I propose a separate setting for the ability to drag

**Comment by neno1978** [comment]
Yeah applebetas is it 👍👍🏿👍👍🏿
